src/bundle/Resources/public/scss/core/_custom.dropdown.scss

```diff
@@ -8,13 +8,13 @@
 
     &__wrapper {
         position: relative;
+        max-width: calculateRem(680px);
     }
 
     &__selection-info {
         list-style: none;
         padding-left: 0;
         margin: 0 0 calculateRem(10px) 0;
-        width: 50%;
```

Comment:
Will this be ok in other places?

Reply:
It's look like this
![Screenshot from 2020-10-19 14-04-49](https://user-images.githubusercontent.com/24355391/96449012-841bea80-1214-11eb-98d2-f2276307d695.png)
![Screenshot from 2020-10-19 14-01-02](https://user-images.githubusercontent.com/24355391/96449014-84b48100-1214-11eb-9e50-845da753bcbc.png)
![Screenshot from 2020-10-19 14-00-45](https://user-images.githubusercontent.com/24355391/96449017-854d1780-1214-11eb-9c12-c032a4f0c957.png)
![Screenshot from 2020-10-19 14-00-23](https://user-images.githubusercontent.com/24355391/96449019-854d1780-1214-11eb-8a43-93af2cd5f214.png)